src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,437 @@ authors: [oluoluoxenfree, alextait1]
+
+A dangerous trend that has seen more exposure than ever in 2020 is the use of “accessibility overlays”. These widgets promise one step accessibility compliance and more often than not introduce new barriers and make the experience for a disabled user quite challenging. It is important that digital practitioners take ownership over designing and implementing usable interfaces and not try to subvert this process with a quick fix. For more information see Lainey Feingold’s article, [Honor the ADA: Avoid Web Accessibility Quick Fix Overlays](https://www.lflegal.com/2020/08/quick-fix/).
+
+Sadly, year over year, we and other teams conducting analysis such as the [WebAIM Million](https://webaim.org/projects/million/https://webaim.org/projects/million/) are finding little and in some cases no improvement in these metrics. The median overall site score for all lighthouse audit data rose from 73% in 2019 to 80% in 2020. We hope that this 7% increase represents a shift in the right direction. However, these are automated checks and could mean that developers are doing a better job of subverting the rule engine, so we are cautiously optimistic.
+
+Our analysis is based on automated metrics only. It is important to remember that automated testing captures only a fraction of the accessibility barriers that can be present in an interface. Qualitative analysis, including manual testing and usability testing with disabled people are needed in order to achieve an accessible site or application. 
+
+We've split up our most interesting insights into five categories: 
+
+1. Ease of reading, 
+2. Media on the web, 
+3. Ease of page navigation, 
+4. Assistive technologies on the web, 
+5. Accessibility of form controls.
+
+## Ease of reading
+
+Making content as simple and clear to read as possible is an important aspect of web accessibility. Being unable to read the content of a page prevents a user from being able to complete tasks on websites. There are many aspects of a web page that make it easier or harder to read, including:
+
+### Color Contrast
+
+The higher the page contrast, the easier it is for people to view text-based content. People who may have difficulties viewing low context content include those with color vision deficiency, people with mild to moderate vision loss, and those with contextual difficulties viewing the content, such as glare on screens in bright light. Unfortunately, only 21.06% of sites were found to have sufficient color contrast. Which is a decrease from last year's already abysmal 22%.
```

Comment:
"People who may have difficulties viewing low ~~context~~ content"

_contrast_

Reply:
@aardrian can you edit your comments to use the "suggestion" feature, which would make it quick and easy to apply your edits?
```suggestion
The higher the page contrast, the easier it is for people to view text-based content. People who may have difficulties viewing low contrast content include those with color vision deficiency, people with mild to moderate vision loss, and those with contextual difficulties viewing the content, such as glare on screens in bright light. Unfortunately, only 21.06% of sites were found to have sufficient color contrast. Which is a decrease from last year's already abysmal 22%.
```

![image](https://user-images.githubusercontent.com/1120896/101567070-27c88200-399e-11eb-9187-1b3dfe4f3d44.png)

Reply:
Poop. Yeah, lemme see if I can go back and re-do.